PowerUp/app/src/main/java/powerup/systers/com/MainActivity.java

```diff
@@ -12,6 +14,15 @@
 	protected void onCreate(Bundle savedInstanceState) {
 		super.onCreate(savedInstanceState);
 		setContentView(R.layout.activity_main);
+		SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(getBaseContext());
+		boolean previouslyStarted = prefs.getBoolean(getString(R.string.pref_previously_started), false);
+		if(!previouslyStarted) {
```

Comment:
Tiny comment: leave a space after if: `if (!previouslyStarted)`
You can select all your code and do `Code->Reformat Code` to format all your code with the same style